idaes/core/solvers/tests/test_petsc.py

```diff
@@ -25,6 +25,32 @@
 from idaes.core.solvers import petsc
 from idaes.core.solvers.features import dae
 
+from pyomo.environ import *
+from pyomo.dae import *
```

Comment:
Looks like you made these unnecessary (good! Namespaces want to be clean!) but forgot to remove them